core/services/keeper/registry_synchronizer_process_logs.go

```diff
@@ -58,7 +58,7 @@ func (rs *RegistrySynchronizer) handleUpkeepCanceledLogs(done func()) {
 		}
 		broadcast, ok := i.(log.Broadcast)
 		if !ok {
-			rs.logger.Errorf("invariant violation, expected log.Broadcast but got %T", broadcast)
+			rs.logger.AssumptionViolationf("expected log.Broadcast but got %T", broadcast)
```

Comment:
I'm thinking that `invariant violation` was actually appropriate for these, but I guess just `critical` level since panicing would change the existing logic.

Reply:
The current AssumptionViolation wraps an error not a panic atm IIRC